endpoints/openrtb2/video_auction.go

```diff
@@ -198,6 +201,12 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 			labels.CookieFlag = pbsmetrics.CookieFlagYes
 		}
 	}
+	if labels.PubID == "" && bidReq.Site != nil && bidReq.Site.Publisher != nil {
```

Comment:
I think we should move this if condition in the above else condition on L196 because say if `req.App` wasn't nil then it's unlikely that `req.Site` will also not be nil. So then in that case we shouldn't need to check again for `req.Site` too. After the if/else, we should just check `if labels.PubID == ""` and set it to `UNKNOWN`